run_tests.sh

```diff
@@ -0,0 +1,7 @@
st,RDFDatetimeTest,FilehashTest,XoredSearchingTest,GRRFuseTest,ClientCommsTest,FilterTests,AFF4SparseImageTest,ByteSizeTest,OSXEnumerateRunningServicesTest,TimelineTest,TestTimelines,TestPackedVersionedCollection,VFSDirectoryTest,FindTest,RekallTests,RDFIntegerTest,TestExecutePython,MethodTest,EmailPluginTest,PackedVersionedCollectionCompactorTest,OSXMemoryTest,HashFileStoreTest,ArtifactFlowTest,AFF4ObjectLabelsListTest,TestExports,AuthTest,ConfigLibTest,GrepTests,GrrKbTest,TestArtifactCollectors,TriggersTest,UsersTest,SshdConfigTest,TestLinuxMemory,DurationTest,ClientURNTests,SqliteDataStoreTest,DataStoreTest,AFF4ObjectLabelTest,TestCheckFlows,TDBDataStoreBenchmarks,ThreadPoolTest,AFF4GRRTest,ChromeHistoryTest,GetClientStatsActionTest,TestSearchFileContent,TestRegistryFinderFlow,AFF4LabelsIndexTest,ListProcessesTest,Ipv6UtilsTest,RDFStructsTest,BuildTests,TestPlistFlows,FileStoreHashTest,ApiConfigRendererTest,IEHistoryTest,RunningStatsTest,AdministrativeFlowTests,YamlConfigTest,RDFValueBaseTest,RDFBytesTest,FileExportPluginTest,TestCopyPathToFile,SqliteDataStoreCSVBenchmarks,RDFValueBenchmark,TDBDataStoreTest,TestLinuxMemoryAnalysis,StoreTests,DataStoreBenchmarks,HintsTests,RekallResponseConverterTest,WindowsRegistryParserTest,ClientUtilsTest,GRRTempFileTestDirectory,EmptyActionTest,FakeDataStoreBenchmarks,AverageMicroBenchmarks,TestListVolumeShadowCopies,WindowsPersistenceMechanismsParserTest,HashDigestTest,WMIParserTest,CollectionExportPluginTest,StatsTests,MasterTest,ApiAff4RendererTest,SendEmailTests,TestClientInterrogate,TestExportHuntResultsFilesAsArchive,RDFURNTest,LinuxSoftwareParserTest,ArtifactParserTest,TestFilesystem,HuntTest,BatchConverterTest,TestArtifactCollectorsInteractions,OSXQuarantineTest,CollectorTest,CollectionFilesExportPluginTest,StandardHuntTest,UserTests,TestSearchFileContentWithFixture,TestCollections,ArtifactParserTests,DarwinPersistenceMechanismsParserTest,RDFValueCollectionApiObjectRendererTest,WebHistoryFlowTest,HTTPDataStoreTest,EmbeddedRDFValueTest,DataAgnosticExportConverterTest,GRRSeleniumTest,SqliteDataStoreBenchmarks,TestSystemRootSystemDriveFallbackFlow,OutputpluginsTest,ConfigFileTest,FlowTest,RegistryFlowTest,ExportTest,MatchMethodTests,ResultSetTest,FakeDataStoreTest,HintDefinitionTests,TestAdministrativeFlows,GRRTempFileTestFilename,FilestoreStatsCronFlowTest,ForemanTests,ReportsTest,GRRBaseTest,TestEndToEndTestFlow,DictTest,TestArtifactCollectorsRealArtifacts,KnowledgeBaseUserMergeTest,HandlerTests,NetstatTest,BasicContextTests,ArtifactHandlingTest,ObjcTest,TestWindowsMemory,WindowsActionTests,GrepTest,SystemCronFlowTest,UtilsTest,DataStoreCSVBenchmarks,CheckLoaderTests,TDBDataStoreCSVBenchmarks,ArtifactKBTest,MultiShardedQueueManagerTest,ConfigActionTest,SearchTest,RegistryVFSTests,OSXDriverTests,RekallTestSuite,GeneralFlowsTest,FlowStateTest,TestCryptoTypeInfos,AFF4Benchmark,GRRArtifactTest,GRRFuseDatastoreOnlyTest,TestOSXFileParsing,AFF4SymlinkTest,FlowTestsBaseclass,ConditionTest,RekallVADParserTest
```

Comment:
I can see your point but my point is that hiding a test is the same as making it red - it is failing so you disable it. Therefore incoming PRs may very well be breaking it but you dont know. Its the head in the sand approach. I think its fair to require a new PR to not break tests and when we get to a green state this will be easier to see but disabling tests is not really helping here.

What exactly is the goal of this CL? If the goal is to get travis build/testing working then it means all tests must pass. Since I know that the test suite passes when run on their own I was hoping this CL can get it to run on travis completely. If you are saying that not all tests are passing then it means this CL is not complete and we should wait until its finished. I dont see the point of submitting a half finished effort just to have that undone in the next CL. Lets get it finished before we use it.

Disabling tests feels like an attempt to get something to be submitted without it being finished. It feels sloppy and there is no reason for it other than shortness of time. If indeed you suggest we adopt a policy of not accepting external PRs which break the build (i.e. they introduce bugs that affect existing tests) that means that those PRs are not finished either. Do you suggest we accept PRs that say - look at this new feature. I know it breaks some tests but i will fix those in the next PR. Clearly not.

Submitting a half working travis test environment is no different. Not all the tests are run so we dont know if its going to work at all.
It is concerning that tests are being killed due to OOM - does this mean travis as a platform is too small for us to run on? If this is the case we dont necessarily want to invest more time in it. This is why we want to see that all tests pass before we commit to using travis at all.